frontend/src/Frontend/UI/Dialogs/AddVanityAccount.hs

```diff
@@ -199,10 +199,7 @@ vanityAccountCreateSubmit model dAccount chainId result nodeInfos = Workflow $ d
   pb <- getPostBuild
   resp <- performLocalRead (model ^. network) $ [req] <$ pb
   let localOk = fforMaybe resp $ \case
-        -- Generates a 'Pattern match has inaccessible right hand side' warning. Not sure why.
-        [(_, t)]
-          | Just (_, PLiteral (LString "Write succeeded")) <- justThat t
```

Comment:
@tomsmalley let me know if this was actually meant to be `justThere`

Reply:
yup, still generates the warning though. I just pushed the extra case to this branch